datasets/za/pmg_legislators/crawler.py

```diff
@@ -0,0 +1,250 @@
+from typing import Dict, Optional, Set
+from followthemoney.helpers import post_summary
+from zavod import Context
+from zavod import helpers as h
+import re
+from urllib.parse import unquote
+from rigour.ids.wikidata import is_qid
+from zavod.logic.pep import categorise
+
+# add headers to the request
+
+PHONE_SPLITS = [",", "/", "(1)", "(2)", "(3)", "(4)", "(5)", "(6)", "(7)", "(8)"]
+PHONE_REMOVE = re.compile(r"(ex|ext|extension|fax|tel|\:|\-)", re.IGNORECASE)
+
+# List of regex patterns for positions of interest
+POSITIONS_OF_INTEREST = [
+    re.compile(r"National\s+Legislature", re.IGNORECASE),
+    re.compile(r"National\s+Assembly", re.IGNORECASE),
+    re.compile(r"National\s+Council of Provinces", re.IGNORECASE),
+    re.compile(r"Minister of", re.IGNORECASE),
+    re.compile(r"Provincial\s+Legislature", re.IGNORECASE),
+    re.compile(r"Member of the Provincial Legislature", re.IGNORECASE),
+    re.compile(r"Member of the Executive Committee", re.IGNORECASE),
+    re.compile(r"National\s+Executive", re.IGNORECASE),
+]
```

Comment:
Currently

```
grep '"Position"' data/datasets/za_pmg_legislators/entities.ftm.json | jq .properties.name|sort | uniq -c | sort -n
```

shows me

```
   1   "Chairperson Of The National Council Of Provinces of the National Council of Provinces"
   1   "Deputy Minister Of ...
...
  1   "Deputy Minister Of ...
   1   "Member Of The Executive Committee: ...
...
   1   "Member Of The Executive Committee: Works of the Free State Provincial Legislature"
   1   "Member Of The Provincial Legislature of the Free State Provincial Legislature"
...
   1   "Member Of The Provincial Legislature of the Western Cape Provincial Legislature"
   1   "Minister Of Agriculture, Forestry And Fisheries of the National Executive"
...
   1   "Minister Of Women, Youth, Children And People With Disabilities of the National Executive"
   1   "Party Leader In Provincial Legislature of the Congress of the People (COPE)"
...
   1   "Party Leader Of Provincial Legislature of the Northern Cape Provincial Legislature"
   1   "Permanent Deputy Chairperson Of The National Council Of Provinces of the National Council of Provinces"
   1   "Provincial Legislature Member of the ANC Constituency Office (102): Pretoria Central"
...
   1   "Provincial Legislature Member of the Gauteng Provincial Legislature"
```

So we're not getting any members of the national assembly or normal members of the NCOP